src/domain/Lotto.js

```diff
@@ -0,0 +1,63 @@
+import { isExpectedArrayLength } from "../utils/checkLength.js";
+import { generateRandomNumbers } from "../utils/generateRandomNumbers.js";
+import { isOverlapped } from "../utils/isOverlapped.js";
+import { sortNumbersAscend } from "../utils/sortNumbersAscend.js";
+import LottoNumber from "./LottoNumber.js";
+import { ERROR_MESSAGE } from "../error/ErrorMessage.js";
+import CustomError from "../error/CustomError.js";
+class Lotto {
```

Comment:
이 사이에는 가독성을 위해 공백이 있는 것이 일반적이에요. 수동으로 정렬하지 않고 eslint, prettier 같은 도구를 사용해 가독성을 높여보면 어떨까요?

```suggestion
import CustomError from "../error/CustomError.js";

class Lotto {
```

Reply:
eslint 세팅하여 적용해보겠습니다!